locales/cy/translations.json

```diff
@@ -1,4 +1,8 @@
 {
     "startTitle" : "Partneriaethau Cyfyngedig yw hyn",
-    "errorStartAddressMissing" : "Rhowch gyfeiriad"
+    "errorStartAddressMissing" : "Rhowch gyfeiriad",
+
+    "pageNotFound_Title" : "WELSH - Page Not Found",
```

Comment:
Just added a couple of Welsh texts in order to check the language switching on the new pages is working.